seed_services_cli/stage_based_messaging.py

```diff
@@ -167,6 +167,37 @@ def messages_import(ctx, csv, json):
             api.create_message(msg)
 
 
+@click.option(
+    '--csv', type=click.File(file_open_mode),
+    help=('CSV file with columns for the endpoint'))
+@click.option(
+    '--json', type=click.File(file_open_mode),
+    help=('JSON objects, one per line for the endpoint'))
+@click.pass_context
+def messages_update(ctx, csv, json):
+    """ Send message updates to the Stage Based Messaging service.
+        binary_content fields should refer to filename in the current folder
+    """
+    if not any((csv, json)):
+        raise click.UsageError("please specify either --csv or --json")
+    api = get_api_client(ctx.obj.stage_based_messaging.api_url,
+                         ctx.obj.stage_based_messaging.token)
+    if csv:
+        for msg in messages_from_csv(csv):
+            if msg["binary_content"] is not None and \
+                    msg["binary_content"] != "":
+                msg = create_binarycontent(api, msg)
+            click.echo("Updating message to messageset %(messageset)s." % msg)
+            api.update_message(msg)
```

Comment:
I think the `update_message` function in the stage based messaging client takes two parameters, the ID of the message, and the data to use to update the message.